Install-Kite.md

```diff
@@ -3,6 +3,8 @@ layout: page
 title: Install the Kite Command Line Interface
 ---
 
+These are the instructions for downloading the Kite command line interface. The most up-to-date instructions for downloading the current version of the software are always available at [http://kitesdk.org/docs/current/Install-Kite.html](http://kitesdk.org/docs/current/Install-Kite.html).
```

Comment:
I think this should include the version that will be installed so that the "current" link makes a bit more sense.